apps/emqx/test/emqx_mqtt_protocol_v5_SUITE.erl

```diff
@@ -25,6 +25,11 @@
 -include_lib("snabbkaffe/include/snabbkaffe.hrl").
 -include_lib("common_test/include/ct.hrl").
 
+-ifndef(BUILD_WITHOUT_QUIC).
+%% Please our CI
```

Comment:
https://github.com/emqx/emqx/blob/e2af92524f532a3e08a8e9d531d0b8092f2d444f/scripts/ct/run.sh#L378

Reply:
This line in [`run.sh` was changed](https://github.com/emqx/emqx/blob/d52144cd80e3b7a80ad3ef253a1627b697b1ba01/scripts/ct/run.sh#L378) in `master`.  Maybe it's better to drop this flag in this branch as well.

Reply:
I will check and do it in another  PR